packages/components/Atoms/Global/scss/settings/_layout.scss

```diff
@@ -15,3 +15,5 @@ $rpl-layout: (
     'xxxl': rem(1420px)
   )
 ) !default;
+
+$rpl-content-max-width: rem(606px);
```

Comment:
@rubyllamadora could you please add ` !default` in the end so it can be overridden by customer site?